app/assets/stylesheets/components/_callout-box.scss

```diff
@@ -115,3 +115,52 @@
     }
   }
 }
+
+.has-tooltip{
+  white-space:nowrap;
+}
+
+.tooltip-wrapper {
+  position: absolute;
+  visibility: hidden;
+}
+
+.has-tooltip:hover .tooltip-wrapper {
+  visibility: visible;
+  opacity: 0.9;
+}
+
+.tooltip-short {
+  display: block;
+  position: relative;
+  top: -2em;
+  right: 125%;
+  width: 105%;
+  color: $white;
+  background: $ocean-blue;
+  text-align: center;
+}
+
+.tooltip-long{
+  display: block;
+  position: relative;
+  top: 2em;
+  right: 125%;
+  width: 105%;
+  color: $white;
+  background: $ocean-blue;
+  text-align: center;
+}
+
+/* Adds a marker to point above the tooltip */
```

Comment:
is this needed?

Reply:
No.  I removed it in the latest commit and added the ellipses.